bindings/python/pmix.pxi

```diff
@@ -245,15 +245,6 @@ cdef int pmix_load_darray(pmix_data_array_t *array, mytype, mylist:list):
         for item in mylist:
             strptr[n] = strdup(item)
             n += 1
-    elif PMIX_STRING == mytype:
```

Comment:
Why remove support for PMIX_STRING?

Reply:
@rhc54 if you scroll up a bit it was just duplicated for some reason. There were two blocks of the same code for PMIX_STRING.